tools/@aws-cdk/prlint/lint.ts

```diff
@@ -438,7 +438,7 @@ function hasLabel(pr: GitHubPr, labelName: string): boolean {
  */
  function validateTitlePrefix(pr: GitHubPr): TestResult {
   const result = new TestResult();
-  const titleRe = /^(feat|fix|build|chore|ci|docs|style|refactor|perf|test)(\([\w_-]+\))?: /;
+  const titleRe = /^(feat|fix|build|chore|ci|docs|style|refactor|perf|test|revert)(\([\w_-]+\))?: /;
```

Comment:
This check is case sensitive, isn't it? Revert is capitalized in the title.

Reply:
I see that you actually updated the title in the one that you posted earlier but wouldn't we want to not have to make people do that each time for this to pass?

Reply:
yeah I went ahead and updated it.